src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DocumentMapping/DocumentPositionInfo.cs

```diff
@@ -10,4 +11,8 @@ namespace Microsoft.CodeAnalysis.Razor.DocumentMapping;
 /// Represents a position in a document. If <see cref="LanguageKind"/> is Razor then the position will be
 /// in the host document, otherwise it will be in the corresponding generated document.
 /// </summary>
-internal readonly record struct DocumentPositionInfo(RazorLanguageKind LanguageKind, Position Position, int HostDocumentIndex);
+internal record struct DocumentPositionInfo(
```

Comment:
Probably not hugely impactful, but was it deliberate to make this not readonly any more?

Reply:
Accidental, changed it when I switched to explicit property version of this, didn't switch back when I changed back to primary constructor version. Fixing, thanks!